PlaceRouteHierFlow/router/PowerRouter.cpp

```diff
@@ -673,6 +674,37 @@ void PowerRouter::CreatePowerGrid(PnRDB::hierNode& node, PnRDB::Drc_info& drc_in
 
 };
 
+void PowerRouter::UpdatePowerGridLLUR(int Lmetal, int Hmetal){
+
+  auto lower_metal = PowerGrid_Drc_info.Metal_info[Lmetal];
+  auto higher_metal = PowerGrid_Drc_info.Metal_info[Hmetal];
+  int x_grid=-1;
+  int y_grid=-1;
+  if(higher_metal.direct == 1){//horizontal
+     y_grid = higher_metal.grid_unit_y ;
+  }else{// vertical
+     x_grid = higher_metal.grid_unit_x ;
+  }
+
+  if(higher_metal.direct == 1){//horizontal
+     y_grid = higher_metal.grid_unit_y ;
+  }else{// vertical
+     x_grid = higher_metal.grid_unit_x ;
+  }
```

Comment:
Why these lines are repeated twice? I guess one of them should be "lower_metal"?

Reply:
The lower one should be lower_metal.